syntax = "proto3";

package envoymobile.extensions.filters.http.test_accessor;

import "buf/validate/validate.proto";

message TestAccessor {
  string accessor_name = 1 [(buf.validate.field).string.min_len = 1];
  string expected_string = 2 [(buf.validate.field).string.min_len = 1];
}
